boards/blues/cygnet/board.cmake

```diff
@@ -5,6 +5,6 @@ board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
 board_runner_args(jlink "--device=STM32L433CC" "--speed=4000")
 
 # keep first
-include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```

Comment:
unrelated change? this prob needs to be explained in the commit message (better, even: a different commit) if this was intentional since this changes default runner